Packs/CommonScripts/Scripts/ParseEmailFiles/test_data/Unknown_encode_attachment_name.eml

```diff
@@ -0,0 +1,20 @@
+Subject: example
+To: <example@example.com>
+Content-Type: multipart/mixed;
+ boundary="_123456_"
+Date: Mon, 01 Nov 2021 12:00:00 +0000
+X-Library: Indy 9.00.10
+Return-Path: exaple@example.com
+
+--_123456_
+Content-Type: multipart/alternative;
+ boundary="_111111_"
+
+--_123456_
+Content-Type: application/octet-stream;
+	name="04. ���A������ИIƷ�|���υf��.pdf"
+Content-Transfer-Encoding: base64
+Content-Disposition: attachment;
+	filename="04. ���A������ИIƷ�|���υf��.pdf"
+
```

Comment:
Let's relace the content of this file with some example content. We dont know if there is sensitive data there.
Also, is the name of the file the one given to us by the customer?
If so, please ask for their permission to use this in our testing.
Content is an open-source repo and we dont want any sensitive information to be there,